ksql-tools/src/main/java/io/confluent/ksql/tools/PrintMetrics.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.tools;
+
+import sun.tools.jconsole.LocalVirtualMachine;
+
+import javax.management.*;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class PrintMetrics {
+  public static void printHelp() {
+    System.err.println(
+        "usage: PrintMetrics "
+            + "[help] "
+            + "pid=<KSQL PID>");
+  }
+
+  public static class PrintMetricsException extends RuntimeException {
+    public PrintMetricsException(String message) {
+      super(message);
+    }
+  }
+
+  private static void printMetrics(int pid) throws IOException {
+    // get all the jvms running locally and find the one we are interested in
+    Map<Integer, LocalVirtualMachine> jvms = LocalVirtualMachine.getAllVirtualMachines();
+    LocalVirtualMachine jvm = null;
+    if (pid == -1) {
+      for (LocalVirtualMachine j : jvms.values()) {
```

Comment:
I'd propose to use the different variable name here. `i, j`s commonly used for index variables.